docs/installation.rst

```diff
@@ -85,6 +85,12 @@ Start by making the following changes to your ``settings.py`` file.
    AUTHENTICATION_BACKENDS = (
        'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
        # ...
+
+   # Set LOGIN_URL to mozilla_django_oidc's login URL.
```

Comment:
@ajcollins can you maybe rephrase this a bit to indicate that is more of a hint and not a necessary step?

Reply:
@akatsoulas sorry for the delay. Good point. PR updated.